src/screens/create-skill/form.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { View } from 'react-native';
+import * as yup from 'yup';
+
+import { ButtonMode, HeadingLevel, SkillType } from 'src/common/enums';
+import { ICreateSkill } from 'src/common/types';
+
+import {
+  Form,
+  FormField,
+  Heading,
+  Input,
+  MainButton,
+  Select,
+} from 'src/components';
+
+import { getNumericEnumValues } from 'src/common/helpers';
+import styles from './styles';
+
+interface Props {
+  onSubmit: (data: ICreateSkill) => void;
+  onCancel: () => void;
+}
+
+const skillTypeOptions = [
+  {
+    label: 'Hard skills',
+    value: SkillType.HardSkills,
+  },
+  {
+    label: 'Soft skills',
+    value: SkillType.SoftSkills,
+  },
+  {
+    label: 'Other',
+    value: SkillType.Other,
+  },
+];
+
+const validationSchema = yup.object({
+  type: yup
+    .number()
+    .integer()
+    .oneOf(getNumericEnumValues(SkillType))
+    .required('Type must be selected'),
+  name: yup.string().required('Name must not be empty'),
+  description: yup.string(),
+  estimate: yup
+    .number()
+    .typeError('Estimate must be a valid number')
+    .min(0, 'Estimate must be from 0 to 10')
+    .max(10, 'Estimate must be from 0 to 10')
+    .required('Estimate must not be empty'),
+});
+
+const CreateSkillForm: React.FC<Props> = ({ onSubmit, onCancel }) => {
+  return (
+    <Form<ICreateSkill>
+      initialValues={{ type: SkillType.Other }}
```

Comment:
Create DEFAULT_CREATE_SKILL_PAYLOAD : ICreateSkillPayload, add all initial values and put in src/constants.ts